xml/cl.xml

```diff
@@ -1615,11 +1615,17 @@ server's OpenCL/api-docs repository.
         <enum value="0x10DE"        name="CL_FLOAT"/>
         <enum value="0x10DF"        name="CL_UNORM_INT24"/>
         <enum value="0x10E0"        name="CL_UNORM_INT_101010_2"/>
-            <unused start="0x10E1" end="0x10E2" comment="Reserved for cl_ext_yuv_images #722"/>
+        <enum value="0x10E1"        name="CL_UNORM_INT10X6_EXT"/>
+            <unused start="0x10E2" end="0x10E2" comment="Reserved for cl_ext_yuv_images #722"/>
```

Comment:
The use of an enum reserved for cl_ext_yuv_images is expected and correct. Both extensions need the same definition and this extension is going to be merged first.